receiver/flinkmetricsreceiver/metadata.yaml

```diff
@@ -0,0 +1,319 @@
+name: flinkmetricsreceiver
+
+resource_attributes:
+  # These resource attributes are Flinks system scope variables, which contains context information about metrics. These are required to uniquely identify incoming metrics as the same job can run multiple times concurrently. See https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/ops/metrics/#system-scope for more information.
+  host.name:
+    description: The host name.
+    type: string
+  flink.taskmanager.id:
+    description: The taskmanager ID.
+    type: string
+  flink.job.name:
+    description: The job name.
+    type: string
+  flink.task.name:
+    description: The task name.
+    type: string
+  flink.subtask.index:
+    description: The subtask index.
+    type: string
+  flink.resource.type:
+    description: The flink scope type in which a metric belongs to.
+    type: string
+
+attributes:
+  operator_name:
+    description: The operator name.
+    type: string
+  garbage_collector_name:
+    description: The names for the parallel scavenge and garbage first garbage collectors.
+    type: string
+    enum: [ PS_MarkSweep, PS_Scavenge, G1_Young_Generation, G1_Old_Generation ]
+  checkpoint:
+    description: The number of checkpoints in progress, completed or that failed. 
+    type: string
+    enum: [ in_progress, completed, failed ]
+  record:
+    description: The number of records received in, sent out or dropped due to arriving late.
+    type: string
+    enum: [ in, out, late_records_dropped ]
```

Comment:
Could we just call the label `dropped`? Seems like the reason why it was dropped is secondary.

Reply:
would loved to do that. updated